modules/claims_api/app/workers/claims_api/special_issue_updater.rb

```diff
@@ -95,7 +98,7 @@ def claim_from_contention_id(claims, contention_id)
     #
     # @param claim [Hash] BGS claim object
     # @param contention_id [Hash(claim_id:, code:, name:)] Identifier to match existing contention
-    # @param special_issues [Array(String)] List of special issues to append
+    # @param claim.special_issues [Array(String)] List of special issues to append
```

Comment:
I believe these are defining the params for the method below. this should remain `special_issues`